pkg/storaged/fsys-tab.jsx

```diff
@@ -415,13 +426,20 @@ export function mounting_dialog(client, block, mode, forced_options) {
                         opts.push("ro");
                     if (vals.mount_options.never_auto)
                         opts.push("x-cockpit-never-auto");
+                    if (!vals.mount_options.fail)
+                        opts.push("nofail");
+                    if (vals.mount_options.netdev)
+                        opts.push("_netdev");
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-17888-20221107-142335-754c0762-fedora-36-devel//Coverage/lcov/github-pr.diff.gcov.html)